src/DependencyInjection/HtmlSanitizerExtension.php

```diff
@@ -75,6 +76,10 @@ private function registerSanitizers(ContainerBuilder $container, array $sanitize
                 $container->setAlias('html_sanitizer', 'html_sanitizer.'.$name);
             }
 
+            if (Kernel::VERSION_ID > 40200) {
+                $container->registerAliasForArgument('html_sanitizer.'.$name, SanitizerInterface::class, "{$name}Sanitizer");
```

Comment:
Can you remove the suffix? It adds a small convention which is unnecessary IMO.

Reply:
IMHO it's more readable with suffix when you read all dependency variables in class instead of class names, but okay, removed :)

Reply:
IMO you already have the type for that: `SanitizerInterface $default` is enough to understand I think.

I'm merging, thanks!